hack/vagrant/bootstrap.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eo pipefail
+shopt -s nullglob
+
+GOVERSION=1.9.2
+SOURCEPATH=/go/src/github.com/alibaba/pouch
+
+# install git/libncurses5-dev/...
+apt-get update
+apt-get install -y git libncurses5-dev libslang2-dev gettext zlib1g-dev libselinux1-dev debhelper lsb-release pkg-config po-debconf autoconf automake autopoint libtool
```

Comment:
This shell command  is too long. We can use the following script to make it readable

```
apt-get install -y git \
    libncurses5-dev \
    .... \
```